pom.xml

```diff
@@ -46,8 +46,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <workflow.version>1.14</workflow.version>
+        <jenkins.version>1.642.3</jenkins.version>
```

Comment:
Ok, I think the answer to my second question is because Pipeline 2.0 requires that baseline.